cogs/admin.py

```diff
@@ -74,23 +71,35 @@ async def sharedservers(self, ctx, *, user: converters.GlobalUser):
                 else:
                     page.description += f"\n{guild}"
             page.set_footer(text="Use the reactions to flip pages.")
-            all_pages.append(page)
+            all_pages.append(page.to_dict())
         if len(all_pages) == 1:
-            embed = all_pages[0]
+            embed = discord.Embed.from_dict(all_pages[0])
             embed.set_footer(text=discord.Embed.Empty)
             await ctx.send(embed=embed)
             return
-        paginator = Paginator(length=1, entries=all_pages, use_defaults=True, embed=True, timeout=120)
-        await paginator.start(ctx)
+        await self.bot.create_reaction_menu(ctx, all_pages)
+
+    async def invite_guild(self, guild_id):
+        guild = await self.bot.get_guild(guild_id)
+        if not guild:
+            return
+        try:
+            invite = (await guild.invites())[0]
+        except (IndexError, discord.Forbidden):
+            try:
+                invite = await self.bot.http.create_invite((await guild.text_channels())[0].id, max_age=120)
```

Comment:
While working on this part, using the create_invite method of the channel did not work.

Reply:
That doesn't help. You need to provide error logs and any additional information.

Reply:
```py
Traceback (most recent call last):
  File "/modmail/cogs/owner.py", line 62, in _eval
    ret = await func()
  File "<string>", line 2, in func
  File "/usr/local/lib/python3.9/site-packages/discord/abc.py", line 751, in create_invite
    return Invite.from_incomplete(data=data, state=self._state)
  File "/usr/local/lib/python3.9/site-packages/discord/invite.py", line 333, in from_incomplete
    channel = guild.get_channel(channel_id) or channel
AttributeError: 'coroutine' object has no attribute 'get_channel'
```
I felt it made more sense to use the http method instead of having to monkey-patch another file.

Reply:
Please go with the patching. We should avoid accessing `bot.http` unless absolutely necessary.